packages/@okta/i18n/src/properties/login.properties

```diff
@@ -803,7 +803,7 @@ oie.okta_verify.totp.title = Enter a code
 oie.okta_verify.totp.enterCodeText = Enter code from Okta Verify app
 oie.okta_verify.push.title = Get a push notification
 oie.okta_verify.push.sent = Push notification sent
-oie.okta_verify.push.send = Send push notification
+oie.okta_verify.push.resend = Resend push notification
```

Comment:
when we do the i18n translations this key name change gets taken care of ? @evelynteo-okta